src/applications/disability-benefits/686c-674/config/address-schema.js

```diff
@@ -279,7 +286,7 @@ export const addressUISchema = (
               type: 'string',
               title: 'City',
               minLength: 1,
-              maxLength: 100,
+              maxLength: 30,
```

Comment:
We may need to also add string truncation in the submit transformer (`customTransformForSubmit`) to fix any save-in-progress data